FastImageCache/FastImageCache/FastImageCache/FICImageTable.m

```diff
@@ -233,10 +233,6 @@ - (instancetype)initWithFormat:(FICImageFormat *)imageFormat imageCache:(FICImag
     return self;
 }
 
-- (instancetype)init {
```

Comment:
That's weird, This line shouldn't be included in that commit. Will fix that in a moment. This line is removed in my other PR  #128